data/wanderer/wanderers start.txt

```diff
@@ -633,6 +658,53 @@ event "Sich'ka'ara restored"
 		add fleet "Wanderer Defense" 2000
 
 
+event "unfettered: outcast"
+	planet "Firelode"
+		remove outfitter "Mera's Lab"
+	planet "Darkcloak"
+		remove outfitter "Unfettered"
+	government "Hai (Unfettered)"
+		"bribe" .04
+	clear "license: True Unfettered"
+
+mission "Unfettered: Regrets"
+	landing
+	invisible
+	to offer
+		not "salary: Unfettered"
+		has "Wanderers: Defend Vara Ke'sok: done"
+	on offer
+		event "unfettered: challengers leave"
+	on enter "Ehma Ti" "Wah Yoot" "Hi Yahr"
+		event "unfettered: outcast"
+		outfit "True Unfettered License"
+		dialog
+			`The instant you enter the system, you receive a priority message from the local authorities. "<last>, because of your recent involvements with the aliens up north, we have decided to stop offering you important missions."`
+			`	Since this is a recorded message you cannot respond, but even if you could they would probably just ignore you. This clearly means that your license has no value anymore to them.`
```

Comment:
```suggestion
			`	Since this is a recorded message you cannot respond, but even if you could they would probably just ignore you. You assume that your Unfettered license no longer has any value to them.`
```